privacy_partner_report/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+======================
+Privacy Partner Report
+======================
+
+This module helps a company to identify all the transactions that a specific
+partner is involved in, with the possibility to export the associated data.
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to menu of ``Data Protection > Reports > Partner Report``
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/263/9.0
+
+Known issues / Roadmap
+======================
+
+* The XLSX works only if https://github.com/odoo/odoo/pull/24964 is merged.
```

Comment:
It seems to be wrong now, although a note to remove the workarounds when that is merged would still be good.

BTW, I think you should use the split readme system instead.

Reply:
Same comment as before.

Reply:
But it worked when I tested it... :thinking: